bears/go/GoErrCheckBear.py

```diff
@@ -0,0 +1,49 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='errcheck',
+        output_format='regex',
+        output_regex=r'(?P<filename>[^:]+):(?P<line>\d+):'
+                     r'(?P<col>\d+)\s*(?P<message>.*)',
+        result_message="This function call has an unchecked error.")
+class GoErrCheckBear:
+
+    """
+    Checks the code for all function calls that have unchecked errors.
+    GoErrCheckBear runs ``errcheck`` over each file to find such functions.
+
+    For more information on the analysis visit <https://github.com/kisielk/errcheck>.
+    """
+    LANGUAGES = "Go"
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         ignore: str='',
+                         ignorepkg: str='',
```

Comment:
those are lists, huh? Please use `typed_list(str)` and concatenate them again so coala does the list verification and parsing.